lib/models/account/user.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:coffeecard/generated/api/coffeecard_api.swagger.swagger.dart';
+import 'package:equatable/equatable.dart';
+
+class User extends Equatable {
+  final int id;
+  final String name;
+  final String email;
+  final bool privacyActivated;
+  final int? programmeId;
+
+  const User({
+    required this.id,
+    required this.name,
+    required this.email,
+    required this.privacyActivated,
+    this.programmeId,
+  });
+
+  User.fromDTO(UserDto dto)
```

Comment:
I think it would be more suitable if the Account Repository returned the user object you needed instead of mapping it to another object outside the repository.

Reply:
`User.fromDTO` is now only called inside `AccountRepository`